src/TyTTP/HTTP/Producer/JSON.idr

```diff
@@ -0,0 +1,20 @@
+module TyTTP.HTTP.Producer.JSON
+
+import Data.Buffer.Ext
+import TyTTP
+import TyTTP.HTTP
+
+export
+json :
```

Comment:
This implementation is unsafe, the String parameter can be anything, there is no guarantee that it is a real json.

This implementation can help though, if it were called as `unsafeJson` and it would be placed into tyttp package's `TyTTP.HTTP.Producer`module.

In this tyttp-json package, the `json` function should rely on the `ToJSON` interface from the `json` package.
If you would change the implementation here to accept `ToJSON a` as parameter and not `String`, then it would be much nicer.

Reply:
Ah good catch! Just updated now 👍